src/Context/WebApiContext.php

```diff
@@ -126,9 +137,19 @@ public function iSendARequestWithValues($method, $url, TableNode $post)
         $bodyOption = array(
           'body' => json_encode($fields),
         );
-        $this->request = $this->getClient()->createRequest($method, $url, $bodyOption);
-        if (!empty($this->headers)) {
-            $this->request->addHeaders($this->headers);
+
+        if (version_compare(ClientInterface::VERSION, '6.0', '>=')) {
+            $this->request = new Request($method, $url, array(), $bodyOption['body']);
```

Comment:
Was not sure because of the `!empty($this->headers)` condition.

Is `headers` attribute always an array?